packages/apidom-converter/test/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-scheme-type/index.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'node:path';
+import { mediaTypes as openAPI31MediaTypes } from '@swagger-api/apidom-parser-adapter-openapi-json-3-1';
+import { mediaTypes as openAPI30MediaTypes } from '@swagger-api/apidom-parser-adapter-openapi-json-3-0';
+import { AnnotationElement, includesClasses, toJSON } from '@swagger-api/apidom-core';
+import { assert, expect } from 'chai';
+
+import convert from '../../../../../src';
+
+describe('converter', function () {
+  context('strategies', function () {
+    context('openapi-3-1-to-openapi-3-0-3', function () {
+      context('security-scheme-type', function () {
+        specify(
+          'should remove SecurityScheme object if it has "mutualTLS" type',
+          async function () {
+            const fixturePath = path.join(__dirname, 'fixtures', 'security-scheme-type.json');
+            const convertedParseResult = await convert(fixturePath, {
+              convert: {
+                sourceMediaType: openAPI31MediaTypes.findBy('3.1.0', 'json'),
+                targetMediaType: openAPI30MediaTypes.findBy('3.0.3', 'json'),
+              },
+            });
+
+            expect(toJSON(convertedParseResult.api!, undefined, 2)).toMatchSnapshot();
+          },
+        );
+
+        specify('should create ERROR annotation', async function () {
+          const fixturePath = path.join(__dirname, 'fixtures', 'security-scheme-type.json');
+          const convertedParseResult = await convert(fixturePath, {
+            convert: {
+              sourceMediaType: openAPI31MediaTypes.findBy('3.1.0', 'json'),
+              targetMediaType: openAPI30MediaTypes.findBy('3.0.3', 'json'),
+            },
+          });
+          const annotations = Array.from(convertedParseResult.annotations);
+          const annotation = annotations.find((a: AnnotationElement) =>
+            a.code?.equals('mutualTLS'),
+          );
+
+          assert.isDefined(annotation);
+          assert.lengthOf(annotations, 2);
+          assert.isTrue(includesClasses(['error'], annotation));
+        });
+      });
+    });
+  });
```

Comment:
We need additional test for attaching source maps to annotations. Pseudo code shows how to achieve that:

```js
specify('should attach source maps annotation', async function () {
  const fixturePath = path.join(__dirname, 'fixtures', 'security-scheme-type.json');
  const convertedParseResult = await convert(fixturePath, {
    parse: {
      parserOpts: { sourceMap: true },
    },
    convert: {
      sourceMediaType: openAPI31MediaTypes.findBy('3.1.0', 'json'),
      targetMediaType: openAPI30MediaTypes.findBy('3.0.3', 'json'),
    },
  });
  const annotations = Array.from(convertedParseResult.annotations);
  const annotation = annotations.find((a: AnnotationElement) => a.code?.equals('mutualTLS'));
  
  toValue(annotation.meta.get('sourceMap')); // assertions  
});

```